### PR TITLE
Added 'h' and 's' options for hue and saturation, added tamper to AC01353010G

### PR DESCRIPTION
**Pull request by jasperro**
This allows for home assistant support of hsv

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -431,9 +440,10 @@ const converters = {
                 const result = await converters.light_color.convertSet(entity, key, value, meta);
                 if (result.state) {
                     result.state.color_temp = utils.xyToMireds(result.state.color.x, result.state.color.y);
+                } else if (value.hasOwnProperty('hue') && value.hasOwnProperty('saturation')) {
+                    result.state = {color: {h: value.hue / 65535 * 360, s: value.saturation / 2.55}};
+                    return result;
                 }
-
-                return result;
```
Comment (Koenkk) [inline converters/toZigbee.js]: Why has this been removed? I think it still required for https://github.com/Koenkk/zigbee-herdsman-converters/pull/841/files#diff-3e0c275bd41af699153bf59d8dda4a2aR442

**Comment by Koenkk** [comment]
Note to self; after this merge: https://github.com/Koenkk/zigbee2mqtt/pull/2603

File: converters/toZigbee.js (hunk 1 above)
Comment (jasperro) [reply converters/toZigbee.js]: It shouldn't have been, I added it back now!

**Comment by Koenkk** [comment]
Thanks for the updates, I will merge it once im back from holiday.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**